tools/toolchain_info.bzl

```diff
@@ -0,0 +1,27 @@
+# look at the toolchains registered in the workspace file with nodejs_register_toolchains
+
+# the name can be anything the user wants this is just added to the target to create unique names
+# the order will match against the order in the TOOLCHAIN_VERSION list.
+TOOLCHAINS_NAMES = [
+    "node14",
+    # TODO: enable node16 for unit tests
```

Comment:
Is this because of build-angular tests?

Reply:
@jbedard  ^

Reply:
@alan-agius4 it seems to be passing fine now (2 builds in a row) so I've enabled it again 🤷